### ISSUE TITLE
[marketplace] Markdown link parsing

### ISSUE THREAD

**Issue by usernamehw** [microsoft/vscode#108026]
https://marketplace.visualstudio.com/items?itemName=usernamehw.errorlens

Having a link with description in README.md:

```md
* [OnClick event on Gutter #5455](https://github.com/microsoft/vscode/issues/5455)
```

It transforms it into 2 links, one of which is broken: 

https://github.com/usernamehw/vscode-error-lens/issues/5455

**Comment by fiveisprime** [comment]
@prashantvc, is this already being tracked?

**Comment by prashantvc** [comment]
No, it is not tracked. I have created an issue in the marketplace repo to track it
